### DISCUSSION TITLE
A simple example for use Zigbee on esp32-h2 for Arduino IDE

### DISCUSSION THREAD

**Discussion by Stefa23Lan** [espressif/arduino-esp32#9032] [Q&A]
It is very difficult, for me, to find a simple example for use Zigbee on esp32-h2 for Arduino IDE. If you have this, thanks to paste here.
I find an example HA-on-off-light for vscode but not for Arduino IDE.
https://github.com/espressif/esp-idf/blob/master/examples/zigbee/light_sample/HA_on_off_switch/main/esp_zb_switch.c
And Vscode with esp-idf 5.2, on my computer, don't see my esp32-h2. But i can run BlinkRGB on Arduino IDE.

**Comment by VojtechBartoska** [comment]
Hi, ported examples are here: https://github.com/espressif/arduino-esp32/pull/9024

**Reply by Stefa23Lan** [reply]
Thank you very much, I'll edulate that. ;)
